specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/capacityReservation.json

```diff
@@ -756,6 +777,10 @@
           "$ref": "#/definitions/CapacityReservationGroupInstanceView",
           "readOnly": true,
           "description": "The capacity reservation group instance view which has the list of instance views for all the capacity reservations that belong to the capacity reservation group."
+        },
+        "sharingProfile": {
+          "$ref": "#/definitions/ResourceSharingProfile",
+          "description": "A list of all subscriptions that the capacity reservation group is shared with. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
```

Comment:
Specifies the settings to enable sharing across subscriptions for the capacity reservation group resource. Pls. keep in mind the capacity reservation group resource generally can be shared across subscriptions belonging to a single azure AAD tenant or cross AAD tenant if there is a trust relationship established between the AAD tenants. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details.